pkg/detectors/replicate/replicate.go

```diff
@@ -0,0 +1,85 @@
+package replicate
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"replicate"}) + `\b(r8_[0-9A-Za-z-_]{37})\b`)
```

Comment:
Is the "replicate" prefix necessary when their token has a defined pattern? I just searched and found numerous examples of replicate tokens that would be missed by including the PrefixRegex.